src/CMakeLists.txt

```diff
@@ -87,6 +87,7 @@ set_target_properties(gpuarray PROPERTIES
   COMPILE_FLAGS "-DGPUARRAY_BUILDING_DLL -DGPUARRAY_SHARED"
   INSTALL_NAME_DIR ${CMAKE_INSTALL_PREFIX}/lib
   MACOSX_RPATH OFF
+  VERSION 0.0
```

Comment:
Normally this automatically handled through shared library versioning.

If you change an interface, you bump the first number, if you add new things you bump the second number.